application/src/main/java/md/maib/retail/CampaignAllInfo.java

```diff
@@ -0,0 +1,34 @@
+package md.maib.retail;
```

Comment:
main package should be md.maib.retail.application

Reply:
but it is in module application.
i should rename and package?